### PR TITLE
use salutation if it exists

### PR DESCRIPTION
**Pull request by gumaerc**
#### Pre-Flight checklist

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
https://github.com/mitodl/ocw-data-parser/issues/136
https://github.com/mitodl/ocw-to-hugo/issues/254

#### What's this PR do?
`ocw-to-hugo` currently has a salutation of "Prof" hard coded for all instructors.  Recently, as noted in the attached issue, `salutation` was added as a property in the OCW Plone JSON exports.  This data is already being included in parsed JSON by `ocw-data-parser`, as the `instructor` objects are simply copied.  This PR uses the `salutation` property if it exists, otherwise it defaults to "Prof."

#### How should this be manually tested?
 - Read the readme if you have never used this utility before
 - Convert `20-109-laboratory-fundamentals-in-biological-engineering-spring-2010`
 - Ensure that the data template at `data/course.json` has a "Dr. Linda Sutliff"
 - Convert another course that does not yet have the salutation field, like `18-06-linear-algebra-spring-2010`
 - Ensure that the instructor is prefixed with "Prof." as it was before

### CODE REVIEW

File: src/data_template_generators.js [hunk 1]
```diff
@@ -26,9 +26,11 @@ const generateDataTemplate = (courseData, pathLookup) => {
...
       : "",
     instructors: (courseData["instructors"] || []).map(instructor => {
-      const name = `${instructor["first_name"]} ${instructor["last_name"]}`
+      const name = `${
+        instructor["salutation"] ? instructor["salutation"] : "Prof."
+      } ${instructor["first_name"]} ${instructor["last_name"]}`
       return {
-        instructor: `Prof. ${name}`,
+        instructor: name,
         url:        helpers.makeCourseInfoUrl(name, "q")
```
Comment (noisecapella) [inline src/data_template_generators.js:34]: It looks like this is now including the salutation as a search term. Is that intentional?

Comment (gumaerc) [reply src/data_template_generators.js:34]: You're right, it's not supposed to be a search term.  I'll correct that.

**Comment by gumaerc** [comment]
@noisecapella all set

**Review by noisecapella — APPROVED** [review]
Works great :+1:

**Pull request merged**